src/Livewire/Order/OrderList.php

```diff
@@ -128,134 +123,57 @@ public function save(): ?false
         return null;
     }
 
-    public function openCreateDocumentsModal(): void
+    protected function getTo(OffersPrinting $item, array $documents): array
```

Comment:
same annotations as in Livewire\Order\Order getTo()

Reply:
√